Marlin/language_en.h

```diff
@@ -602,6 +602,45 @@
 #ifndef MSG_FILAMENTCHANGE
   #define MSG_FILAMENTCHANGE                  _UxGT("Change filament")
 #endif
+#ifndef MSG_FILAMENTLOADUNLOAD
+  #define MSG_FILAMENTLOADUNLOAD              _UxGT("Filament Un/Load")
+#endif
+#ifndef MSG_FILAMENTLOAD
+  #define MSG_FILAMENTLOAD                    _UxGT("Load filament")
+#endif
+#ifndef MSG_FILAMENTLOAD1
+  #define MSG_FILAMENTLOAD1                   _UxGT("Load filament 1")
+#endif
```

Comment:
Only one `MSG_FILAMENTLOAD` is needed. Append `MSG_E0`, `MSG_E1`, etc., in the code if numbers are needed. For example:
```cpp
MENU_ITEM(gcode, MSG_FILAMENT_LOAD MSG_E0, PSTR("M701 T0"));
```